docs/courses/design/brainstorming-jargon.md

```diff
@@ -0,0 +1,40 @@
+# Step 1.4 - What terms or jargon will you define?
+
+Write a list of technical terms, jargon, and acronyms that will be used in the course.
+
+
+## Examples
+
+From a course on experimental design. This has an extensive list of statistical terms.
+
+Randomization, replication, blocking, Latin Square, Greco-Latin Squares, factorial, ANOVA, T-test, F-test, normality, qqplot, variance, type I/II error, null/alternative hypothesis, effect size, factor/categorical variable
+
+
+From a course on clinical trials analysis. This includes both statistical terms and domain-specific terms.
+
+Bias, blinding, randomization, imbalance, covariates, endpoints, power, multiplicity, significance, non-inferiority, equivalence, bioequivalence.
+
+
+## FAQs
+
+### What can I assume that the students will know already?
+
+This depends a lot upon the prerequisites for the course. For example, the introductory statistical courses have to explain what a statistical model is. For most statistical courses, you don't need to do that since they will have one of those courses are as prerequisite.
+
+
+## Good ideas
+
+### Think of synonyms
+
+Often there are several different words for a particular concept. For example, columns in a rectangular dataset may also be referred to as fields or features or variables, depending upon context. Many students may have head one term but not the others, so it can be useful to explain synonyms to them.
+
```

Comment:
It's useful to explain synonyms, but it's _not_ useful to go back-and-forth between them. Consistency in terminology is much more conducive to learning than making students remember six words mean the same thing. I think most instructors do this well, but I've run into a few where they will jump from one term to the next and I find myself lost.

Reply:
fixed